pkg/ui/react-app/src/thanos/pages/stores/StorePoolPanel.tsx

```diff
@@ -35,7 +35,19 @@ export const storeTimeRangeMsg = (validMin: boolean, validMax: boolean): string
 
   return '';
 };
+export const getColorTimeRangeMsg = (validMin: boolean, validMax: boolean): string => {
+  if (!validMin && !validMax) {
+    return "red";
```

Comment:
There are some linting issues (which is why the tests fail), but these can be fixed automatically, just need to run `make react-app-lint-fix`! :)